dags/common/enricher.py

```diff
@@ -63,7 +63,7 @@ def _get_arxiv_categories_from_response_xml(self, xml: ET.Element):
         return list([primary_category] + secondary_categories)
 
     @backoff.on_exception(
-        backoff.expo, requests.exceptions.RequestException, max_time=60, max_tries=3
+        backoff.expo, requests.exceptions.RequestException, max_tries=5
```

Comment:
this is a brute force, we should not keep a connection more than 60, it's too much. Let's just increase the retries.